packages/components/tests/integration/components/hds/form/text-input/field-test.js

```diff
@@ -98,25 +98,29 @@ module('Integration | Component | hds/form/text-input/field', function (hooks) {
       hbs`<Hds::Form::TextInput::Field as |F|>
           <F.Label>This is the label</F.Label>
           <F.HelperText>This is the helper text</F.HelperText>
+          <F.CharacterCount>20/40</F.CharacterCount>
           <F.Error>This is the error</F.Error>
         </Hds::Form::TextInput::Field>`
     );
     assert.dom('.hds-form-field__label').exists();
     assert.dom('.hds-form-field__helper-text').exists();
     assert.dom('.hds-form-field__control').exists();
+    assert.dom('.hds-form-field__character-count').exists();
     assert.dom('.hds-form-field__error').exists();
   });
   test('it does not render the yielded contextual components if not provided', async function (assert) {
     await render(hbs`<Hds::Form::TextInput::Field />`);
     assert.dom('.hds-form-field__label').doesNotExist();
     assert.dom('.hds-form-field__helper-text').doesNotExist();
+    assert.dom('.hds-form-field__character-count').doesNotExist();
     assert.dom('.hds-form-field__error').doesNotExist();
   });
   test('it automatically provides all the ID relations between the elements', async function (assert) {
     await render(
       hbs`<Hds::Form::TextInput::Field @extraAriaDescribedBy="extra" as |F|>
           <F.Label>This is the label</F.Label>
           <F.HelperText>This is the helper text</F.HelperText>
+          <F.CharacterCount>20/40</F.CharacterCount>
```

Comment:
here we're testing the "static" version of the character count (which is already tested in the "field" component). Should we test also (or only) the "dynamic" version, to make sure it works as expected, in relation to the `input` value/lenght? (or we test it only in the `field` component, as suggested above)

Reply:
indeed, using the 'dynamic' version makes more sense for the Field components – updated in 9127638